website/docs/guides/sl-cookbook-recipes.md

```diff
@@ -0,0 +1,648 @@
+- Learn how to define and calculate CAC by dividing the total acquisition cost by the number of new customers.
+
+### Use case
+Track and analyze the cost of acquiring new customers to optimize marketing and sales strategies.
+
+:::tip
+💡 Note that we’re using the [parallel subfolder approach](/best-practices/how-we-build-our-metrics/semantic-layer-7-semantic-structure#files-and-folders) for better organization in large projects, which means you’re creating a semantic model folder containing all your semantic layer code. 
+
+Alternatively, you can use the [one-YAML-per-marts-model approach](/best-practices/how-we-build-our-metrics/semantic-layer-7-semantic-structure#files-and-folders) which puts documentation, data tests, unit tests, semantic models, and metrics into a unified file corresponding to a dbt-modeled mart.
+:::
+
+### Step 1: Define semantic models
+Create a YAML file under the `semantic_models` folder: (`models/semantic_models/sem_customer_acquisition.yml`) to define a semantic model for customer acquisition data.
+
+<File name="models/semantic_models/sem_customer_acquisition.yml">
+
+```yaml
+semantic_models:
+  - name: sem_customer_acquisition
+    description: |
+      This model aggregates customer acquisition data, providing metrics for business to analyze the cost of acquiring new customers.
+    model: ref('fct_customer_acquisition')
+    defaults:
+      agg_time_dimension: date_month
+    entities:
+      - name: date
+        type: primary
+        expr: date_month
+    measures:
+      - name: total_acquisition_cost
+        agg: sum
+        expr: acquisition_cost
+        agg_time_dimension: date_month
+      - name: new_customers
+        agg: sum
+        expr: customer_count
+        agg_time_dimension: date_month
+    dimensions:
+      - name: date_month
+        type: time
+        type_params:
+          time_granularity: month
+      - name: acquisition_channel
+        type: categorical
+```
+
+</File>
+
+### Step 2: Define the CAC metric
+Create another YAML file under the metrics folder: (`models/metrics/sem_customer_acquisition_metrics.yml`) to define the CAC metric.
+
+<File name="models/metrics/sem_customer_acquisition_metrics.yml">
+
+```yaml
+metrics:
+  - name: cac
+    description: "Customer acquisition cost"
+    label: 'CAC'        
+    type: ratio
+    type_params:
+      numerator: total_acquisition_cost
+      denominator: new_customers
+```
+
+</File>
+
+### Step 3: Validate and query metric
+
+1. Confirm and validate the new metric definition in your development by running the following command:
+   
+   ```bash
+   dbt parse
+   ```  
+
+2. Query your metrics to confirm the results are as expected. Run the following command:
+   
+   ```bash
+   dbt sl query --metrics cac --group-by metric_time__year --where "metric_time__year >= '2023'" --order-by metric_time__year
+   ```
+<!-- Cookbook 4 -->
+
+## How to join multiple fact metrics
```

Comment:
hey @patkearns10  - would love your feedback and perspective on this join metrics section if you have time!

Reply:
Couple quick changes, but otherwise looks good to me!